chromium_src/components/sync_device_info/device_info_sync_bridge.cc

```diff
@@ -89,4 +93,28 @@ DeviceInfoSyncBridge::GetAllBraveDeviceInfo() const {
   return list;
 }
 
+void DeviceInfoSyncBridge::RefreshLocalDeviceInfoIfNeeded(
+    base::OnceClosure callback) {
+  const DeviceInfo* current_info =
+      local_device_info_provider_->GetLocalDeviceInfo();
+  if (!current_info) {
+    if (!callback.is_null()) {
+      device_info_synced_callback_list_.push_back(std::move(callback));
+    }
+    return;
+  }
+
+  auto iter = all_data_.find(current_info->guid());
+  if (iter == all_data_.end()) {
```

Comment:
```base::Contains``` seems better, because the ```iter```s content is not used.

Reply:
thanks, fixed in https://github.com/brave/brave-core/compare/e58b4936e0d1630c6add576c64c3bbcc6722bdff..c7a203066e866e89bf5556874f0294e1b262101f